stories/Grid.story.jsx

```diff
@@ -4,47 +4,42 @@ import { withInfo } from "@storybook/addon-info";
 import styled from "styled-components";
 import { Row, Col } from "../src/index";
 
-const OffsetGrid = styled.div`
-  .row {
-    margin-bottom: 10px;
-  }
-`;
 const Box = styled.div`
-  background-color: #04d4bf;
-  height: 50px;
-`;
-
-const BoxBlue = styled.div`
-  background-color: #0000ff;
-  height: 50px;
+  -webkit-box-shadow: 0px 1px 2px 0 rgba(34, 36, 38, 0.15);
+  box-shadow: 0px 1px 2px 0 rgba(34, 36, 38, 0.15);
+  margin: 2px;
+  padding: 1em 1em;
+  border: 1px solid #01a1a1;
+  background-color: #b3e1e5;
+  flex: 1;
 `;
 
-const BoxYellow = styled.div`
-  background-color: #ffff19;
-  padding: 10px;
-  min-height: 50px;
+const BoxWhite = styled.div`
+  -webkit-box-shadow: 0px 1px 2px 0 rgba(34, 36, 38, 0.15);
+  box-shadow: 0px 1px 2px 0 rgba(34, 36, 38, 0.15);
+  margin: 2px;
+  padding: 1em 1em;
+  border: 1px solid #01a1a1;
+  background-color: #fff;
+  flex: 1;
 `;
 
 storiesOf("Grid", module)
   .addWithJSX(
-    "default",
-    withInfo()(() => (
+    "default width",
+    withInfo(
+      "would have the default maximum width of the column which is 960px"
```

Comment:
Use complete sentences please. "The Row would have the default max....960px." or "The Grid" or whatever subject you're referring to.